doc/visualizations.rst

```diff
@@ -8,11 +8,19 @@ Scikit-learn defines a simple API for creating visualizations for machine
 learning. The key feature of this API is to allow for quick plotting and
 visual adjustments without recalculation. We provide `Display` classes that
 expose two methods for creating plots: `from_estimator` and
-`from_predictions`. The `from_estimator` method will take a fitted estimator
-and some data (`X` and `y`) and create a `Display` object. Sometimes, we would
-like to only compute the predictions once and one should use `from_predictions`
-instead. In the following example, we plot a ROC curve for a fitted support
-vector machine:
+`from_predictions`.
+
+The `from_estimator` method generates a `Display` object from a fitted estimator and input data (X, y).
+The `from_predictions` method creates a `Display` object when given the true and predicted values.
+This is useful when predictions should only be computed once.
```

Comment:
```suggestion
The `from_predictions` method creates a `Display` object from true and predicted values, which
is useful when you only want to compute the predictions once.
```

nitpick, I think it would be nice to use the same phrasing, as these 2 method do the same thing, just with different inputs.

Reply:
Done